src/main/java/org/tikv/common/region/RegionStoreClient.java

```diff
@@ -879,7 +879,7 @@ private void rawPutHelper(RawPutResponse resp) {
       throw new TiClientInternalException("RawPutResponse failed without a cause");
     }
     String error = resp.getError();
-    if (error != null && !error.isEmpty()) {
+    if (!error.isEmpty()) {
```

Comment:
it's not related to the pr title.

Reply:
They are related to similar changes in `rawDeleteRangeHelper`